architecture/additional_concepts/storage.adoc

```diff
@@ -449,6 +449,15 @@ Fibre Channel and HostPath persistent volumes do not support mount options.
 ====
 endif::openshift-enterprise,openshift-origin[]
 
+=== Recursive chown
+
+Each time a persistent volume is mounted to a pod, or when a pod restart occurs, the volume's ownership and permissions are recursively changed to match those of the pod. Group read/write access is added to the volume when *chown* is performed on all files and directories in the PV. This enables the process running inside the pod to access the PV's file system.
```

Comment:
@bobfuru Is this update necessary if "persistent volume (PV)" is already in the Overview for https://docs.openshift.com/container-platform/3.11/architecture/additional_concepts/storage.html?

Reply:
It's debatable. FCC suggests defining on first instance (as indicated in my comment) within every module. But most modules as of yet have not been reused as standalone content. So you're likely to see a mix of conventions. 

I actually call it out here because you spell it out in the first sentence and then initialize in the following sentences.